src/main/kotlin/no/nav/familie/ef/søknad/mapper/SkjemaMapper.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.ef.søknad.mapper
+
+import no.nav.familie.ef.søknad.api.dto.søknadsdialog.Arbeidssøker
+import no.nav.familie.kontrakter.ef.søknad.Fødselsnummer
+import no.nav.familie.kontrakter.ef.søknad.SkjemaForArbeidssøker
+import no.nav.familie.kontrakter.ef.søknad.Søknadsfelt
+import org.springframework.stereotype.Component
+import no.nav.familie.kontrakter.ef.søknad.Arbeidssøker as ArbeidssøkerKontrakt
+
+@Component
+object SkjemaMapper {
+
+    fun mapTilKontrakt(arbeidssøker: Arbeidssøker, fnr: String): SkjemaForArbeidssøker {
+        val arbeidssøkerKontrakt = arbeidssøker.toArbeidssøkerKontrakt()
+        return SkjemaForArbeidssøker(fødselsnummer = Søknadsfelt("Fødselsnummer",
+                                                                 Fødselsnummer(fnr)),
+                                     arbeidssøker = arbeidssøkerKontrakt)
+    }
+}
+
+private fun Arbeidssøker.toArbeidssøkerKontrakt(): Søknadsfelt<ArbeidssøkerKontrakt> {
```

Comment:
(Ikke gjort mye Kotlin før, så muligens dumt spørsmål)

Hva oppnår/mister vi ved å utvide Arbeidssøker med denne metoden kontra statisk mapping med Arbeidssøker som input? Ser at vi slipper `arbeidssøker.` foran hvert attributt - det er jo noe :)

Reply:
Her var jeg mest ute etter å slippe "arbeidssøker." ja, så ikke så mye :) Men - vi mapper arbeidssøker i søknad også, så mulig vi burde bruke samme mapping de to stedene?